extensions/commit-message-generator/README.md

```diff
@@ -0,0 +1,3 @@
+# Commit Message Generator
+
+whatthecommit.com
```

Comment:
Could you add a bit more context here since there is no metadata due to no-view command 🙂